sklearn/tests/test_pipeline.py

```diff
@@ -1071,3 +1072,12 @@ def test_make_pipeline_memory():
     assert pipeline.memory is None
 
     shutil.rmtree(cachedir)
+
+
+def test__len__():
+    pipeline = Pipeline([
+        ('vect', CountVectorizer()),
+        ('tfidf', TfidfTransformer()),
```

Comment:
You could avoid the extra import by just putting StandardScaler here

Reply:
Yes changing it now.